packages/kbn-securitysolution-list-hooks/src/use_exception_lists/index.ts

```diff
@@ -43,6 +44,7 @@ export const useExceptionLists = ({
   namespaceTypes,
   notifications,
   showTrustedApps = false,
+  showEventFilters = false,
```

Comment:
Yeah! I'm agree that would be better to filter by what we want instead that filter what we want to skip (event filters list and trusted apps list in this case). 
I'm not sure why this was done in this way. Is it supposed to have more cain of exceptions lists than the current ones? Is this something user can create and then, yes, we should just filter trusted apps and event filter lists in order to avoid loosing data?

Reply:
yeah, I'm not sure either, but we can take that on to another (potential refactor) issue.